src/main/webapp/js/user-page-loader.js

```diff
@@ -95,4 +95,5 @@ function buildUI() {
   setPageTitle();
   showMessageFormIfViewingSelf();
```

Comment:
I think this function call will need to change to match the updated function name.

Reply:
I was having trouble with my code because I didn't have the latest code on my local repository before I started making changes. Could I close this pull request and make a new one? The code I have now is able to merge without conflict.

Reply:
Sure, if that's easier, you can just check out the master branch, and then create a new branch off of it with all your changes.  And you can close and delete this one.